src/components/landingPage/_SignUpComponent.svelte

```diff
@@ -0,0 +1,472 @@
+		</svelte:fragment>
+		<section class="bg-gray-50 dark:bg-gray-900">
+			<div class="flex flex-col items-center justify-center px-6 py-8 mx-auto lg:py-0 w-full">
+				<div class="p-6 space-y-4 md:space-y-6 sm:p-8 flex flex-wrap justify-center w-full">
+					{#await registerPromise}
+						<ProgressRadial
+							class="w-1/2 h-1/2"
+							stroke={100}
+							meter="stroke-primary-500"
+							track="stroke-primary-500/30"
+						/>
+					{:then}
+						<h5 class="h5 text-center">Please enter the code we sent to your email!</h5>
+						<form
+							class="space-y-4 md:space-y-6 group flex flex-wrap justify-center"
+							action="#"
+							novalidate
+						>
+							<label class="label">
+								<input
+									class="input h-6 w-6 md:h-8 md:w-8 lg:h-10 lg:w-10 text-center rounded form-control"
+									title="token1"
+									maxlength="1"
+									type="text"
+									placeholder=""
+									autocomplete="off"
+								/>
+								<input
+									class="input h-6 w-6 md:h-8 md:w-8 lg:h-10 lg:w-10 text-center rounded"
+									title="token2"
+									maxlength="1"
+									type="text"
+									placeholder=""
+									autocomplete="off"
+								/>
+								<input
+									class="input h-6 w-6 md:h-8 md:w-8 lg:h-10 lg:w-10 text-center rounded"
+									title="token3"
+									maxlength="1"
+									type="text"
+									placeholder=""
+									autocomplete="off"
+								/>
+								<input
+									class="input h-6 w-6 md:h-8 md:w-8 lg:h-10 lg:w-10 text-center rounded"
+									title="token4"
+									maxlength="1"
+									type="text"
+									placeholder=""
+									autocomplete="off"
+								/>
+								<input
+									class="input h-6 w-6 md:h-8 md:w-8 lg:h-10 lg:w-10 text-center rounded"
+									title="token5"
+									maxlength="1"
+									type="text"
+									placeholder=""
+									autocomplete="off"
+								/>
+								<input
+									class="input h-6 w-6 md:h-8 md:w-8 lg:h-10 lg:w-10 text-center rounded"
+									title="token6"
+									maxlength="1"
+									type="text"
+									placeholder=""
+									autocomplete="off"
+								/>
+							</label>
+							<ol class="list">
+								<li>
+									{#if errorState}
+										<span class="badge-icon variant-filled-success w-4 h-4"><CrossIcon /></span>
+										<span class="flex-auto">False code, please enter </span>
+									{/if}
+								</li>
+							</ol>
+							<button
+								type="submit"
+								class="w-full btn variant-filled-primary hover:variant-soft-primary dark:hover:variant-soft-primary-dark {true
+									? ''
+									: 'pointer-events-none opacity-50'}"
+								on:click={() => {
+									loading = true;
+								}}
+								on:click={register}
+							>
+								{#if !loading}
+									Verify token
+								{:else}
+									<ProgressRadial width="w-6" />
+								{/if}
+							</button>
+						</form>
+					{:catch error}
+						<p>error {error}</p>
+					{/await}
+				</div>
+			</div>
+		</section>
+	</Step>
```

Comment:
Und Step 4. Ich glaube das ist dann schon deutlich lesbarer. Sonst finde ich dein Sign Up wunderbar

Reply:
Das ganze Modal sieht nur leider auf dem Galaxy Fold echt furchtbar aus xD

Reply:
As I said, Galaxy Fold for the trashcans